core/src/main/scala/ai/h2o/sparkling/backend/external/K8sH2OStatefulSet.scala

```diff
@@ -43,29 +42,21 @@ trait K8sH2OStatefulSet extends K8sUtils {
   }
 
   private def waitForClusterToBeReady(client: KubernetesClient, conf: H2OConf): String = {
-    client
```

Comment:
Isn't there this piece of code to check whether the stateful set exist or not? if not, it throws an exception?

Reply:
According to my understanding, this code is somehow specific to H2O cluster, i.e. all the StatefulSet pods have to be running, but just one of them should be ready.

Reply:
yes, this is check afterwards. But doesn't the below snippet check that the statefulset exist? In other words, this additional check, ensures that pods haven't been created without stateful set.

```
client
      .apps()
      .statefulSets()
      .inNamespace(conf.externalK8sNamespace)
      .withName(conf.externalK8sH2OStatefulsetName)
```

Reply:
It's  nop command